frontend/src/App.js

```diff
@@ -57,26 +57,64 @@ export class App extends React.Component {
 
   handleChange = (change) => {
     switch (change.operation) {
-      case "addElement":
-        this.addElement(change.data);
+      case "addComponent":
+        this.addComponent(change.data);
         break;
       case "moveComponent":
         this.moveComponent(change.data);
         break;
       case "deleteComponent":
         this.deleteComponent(change.data);
         break;
+
```

Comment:
extra line here

Reply:
I left that in intentionally, for visual separation

Reply:
fair enough